docs/extensions/lifecycle.md

```diff
@@ -249,3 +247,36 @@ Based on a project's maturity and stewardship, it may be eligible to use resourc
 | QA	| The `civicrm.org` build-bot runs extension tests for PRs (own repo)	| "Official" projects (regardless of stability)
 | QA	| The `civicrm.org` build-bot runs extension tests for PRs (civicrm-core repo)	| "Official" projects ("Stable" or "Incubation")
 | Support	| The project may have its own space or component on "lab.civicrm.org"	| "Official" projects (regardless of stability)
+
+## Abandoned Extensions {:#abandoned-extensions}
+
+A CiviCRM Extension may be abandoned when the project author is no longer releasing updates and/or is not responding to support requests within 14 days or has expressly stated that they are no longer actively maintaining the project.
+
+The CiviCRM community has two options for managing an abandoned extension.
+
+### Transfer Ownership to a New Maintainer
+
+When an extension author has stopped being responsive:
+* and there are important bugs in the extension
```

Comment:
```suggestion

* and there are important bugs in the extension
```

Reply:
Thanks for reviewing @MikeyMJCO - change has been done.